{
    "url": "https://api.github.com/repos/spack/spack/issues/9769",
    "repository_url": "https://api.github.com/repos/spack/spack",
    "labels_url": "https://api.github.com/repos/spack/spack/issues/9769/labels{/name}",
    "comments_url": "https://api.github.com/repos/spack/spack/issues/9769/comments",
    "events_url": "https://api.github.com/repos/spack/spack/issues/9769/events",
    "html_url": "https://github.com/spack/spack/issues/9769",
    "id": 378862735,
    "node_id": "MDU6SXNzdWUzNzg4NjI3MzU=",
    "number": 9769,
    "title": "glib build fails (both for python2x and 3x)",
    "user": {
        "login": "Sinan81",
        "id": 2924561,
        "node_id": "MDQ6VXNlcjI5MjQ1NjE=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2924561?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Sinan81",
        "html_url": "https://github.com/Sinan81",
        "followers_url": "https://api.github.com/users/Sinan81/followers",
        "following_url": "https://api.github.com/users/Sinan81/following{/other_user}",
        "gists_url": "https://api.github.com/users/Sinan81/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/Sinan81/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/Sinan81/subscriptions",
        "organizations_url": "https://api.github.com/users/Sinan81/orgs",
        "repos_url": "https://api.github.com/users/Sinan81/repos",
        "events_url": "https://api.github.com/users/Sinan81/events{/privacy}",
        "received_events_url": "https://api.github.com/users/Sinan81/received_events",
        "type": "User",
        "site_admin": false
    },
    "labels": [],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [],
    "milestone": null,
    "comments": 13,
    "created_at": "2018-11-08T18:33:26Z",
    "updated_at": "2018-11-13T16:37:20Z",
    "closed_at": "2018-11-13T16:37:20Z",
    "author_association": "CONTRIBUTOR",
    "active_lock_reason": null,
    "body": "### Steps to reproduce the issue\r\n\r\n```console\r\n$ spack install glib \r\n```\r\nin the log we are seeing the following errors:\r\n\r\n````\r\n1831  /usr/bin/install -c -m 644 ./html/up-insensitive.png\r\n1832  /usr/bin/install -c -m 644 ./html/up.png\r\n1833 Traceback (most recent call last):\r\n1834   File \"/usr/bin/gtkdoc-rebase\", line 28, in <module>\r\n1835     from gtkdoc import common, config, rebase\r\n1836   File \"/usr/share/gtk-doc/python/gtkdoc/common.py\", line 31, in <module>\r\n1837     import six\r\n1838 ImportError: No module named six\r\n1839 Makefile:1034: recipe for target 'install-data-local' failed\r\n1840 make[5]: *** [install-data-local] Error 1\r\n1841 make[5]: Leaving directory '/tmp/sinan/spack-stage/spack-stage-9BmR48/glib-2.56.2/docs/reference/glib'\r\n1842 Makefile:776: recipe for target 'install-am' failed\r\n1843 make[4]: *** [install-am] Error 2\r\n1844 make[4]: Leaving directory '/tmp/sinan/spack-stage/spack-stage-9BmR48/glib-2.56.2/docs/reference/glib'\r\n1845 Makefile:484: recipe for target 'install-recursive' failed\r\n1846 make[3]: *** [install-recursive] Error 1\r\n1847 make[3]: Leaving directory '/tmp/sinan/spack-stage/spack-stage-9BmR48/glib-2.56.2/docs/reference'\r\n1848 Makefile:485: recipe for target 'install-recursive' failed\r\n1849 make[2]: *** [install-recursive] Error 1\r\n1850 make[2]: Leaving directory '/tmp/sinan/spack-stage/spack-stage-9BmR48/glib-2.56.2/docs'\r\n1851 Makefile:1279: recipe for target 'install-recursive' failed\r\n1852 make[1]: *** [install-recursive] Error 1\r\n1853 make[1]: Leaving directory '/tmp/sinan/spack-stage/spack-stage-9BmR48/glib-2.56.2'\r\n1854 Makefile:1748: recipe for target 'install' failed\r\n````\r\n[spack-build.out.txt](https://github.com/spack/spack/files/2563044/spack-build.out.txt)\r\n\r\n\r\n### Platform and user environment\r\n\r\nPlease report your OS here:\r\n```commandline\r\n$ uname -a\r\nLinux storm 4.15.0-36-generic #39-Ubuntu SMP Mon Sep 24 16:19:09 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux\r\n$ lsb_release -d\r\nDescription:    Linux Mint 19 Tara\r\n``` \r\nThis error should be also reproduceable on Ubuntu 18.04.1\r\n\r\nI think this is not a major problem as error has to do with the building of the documentation. If I understand correctly, glib is attempting to use already existing gtk documentation in the system.",
    "performed_via_github_app": null
}